SimpleGame/src/main/scala-2.11/edu/gemini/experimental/jluhrs/Timer.scala

```diff
@@ -0,0 +1,17 @@
+package edu.gemini.experimental.jluhrs
+
+/**
+  * Created by jluhrs on 5/19/16.
+  */
+
+// The source of time (can we really control time ?)
+object Timer {
+  // For the sake of simplicity, Time is defined as Double
+  type Time = Double
```

Comment:
Is time just an ever increasing counter used for ordering events or is it a timestamp corresponding to a real-world time?  Either way, why a `Double`?